configs/config_memeooorr.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Memeooorr",
     "agent_id": 43,
-    "hash": "bafybeia34pecf46znvghrbq32trmjwurgxk5ybrslgx6c4xwkmoeqhhsta",
+    "hash": "bafybeicpoldxi2xsjbczckmhdinp3x4rttiz3mtw7slpaqs5zw3pdvzamq",
```

Comment:
This hash is now outdated: bafybeig5h3el4kz3o7epnwtuvzhlojlgcpw3rlkmnqt5xjc6dnyio2afqi is lates one released